src/style.css

```diff
@@ -1 +1,60 @@
-/* Styles go here */
+body {
+  margin: 0;
+  background-color: #e5e5e5;
+}
+
+.nav {
+  margin-right: 50px;
+}
+
+header {
+  display: flex;
+  background-color: #fff;
+  justify-content: space-between;
+  align-items: center;
+  flex-wrap: nowrap;
+}
+
+.logo {
+  margin: 10px 0 10px 50px;
+  height: 40px;
+  width: 40px;
+}
+
+.text {
+  height: 60px;
+  text-align: center;
+  font-size: 12px;
+  text-transform: uppercase;
+  font-weight: bold;
+  font-family: Roboto, sans-serif;
+  color: rgba(0, 0, 0, 1);
+}
+
+.text:hover {
+  color: rgba(0, 172, 210, 1);
+
+}
+
+a {
+  text-decoration: none;
+  display: inline-block;
+}
+
+.list {
+  display: inline-block;
+  padding: 16px;
+  margin: 0;
+  height: 100px;
```

Comment:
`list` cannot have this height, let the content inside `list` dictate its height